sky/packages/sky/lib/src/widgets/ink_well.dart

```diff
@@ -33,11 +35,12 @@ class InkSplash {
     _performance = new ValueAnimation<double>(
       variable: _radius,
       duration: new Duration(milliseconds: (_targetRadius / _kSplashUnconfirmedVelocity).floor())
-    )..addListener(_handleRadiusChange)
-     ..play();
+    )..addListener(_handleRadiusChange);
+
+    // Wait kTapTimeout to avoid creating tiny splashes during scrolls.
+    _startTimer = new Timer(kTapTimeout, _play);
```

Comment:
shouldn't we rely on the gestures to do this? i.e. have the tap gesture recogniser fire onTapStart either after kTapTimeout after onTapDown, OR when it's won, whichever comes first? Otherwise, taps not in lists will be delayed also.